src/modules/proto.haml

```diff
@@ -73,10 +69,13 @@
                         <!-- Other modules. -->
                         </map>
                         ```
-
                     * A team can have a minimum required size with the `min=""` attribute.
                     * A players gamemode can be changed by using a the `<game-mode>adventure</game-mode>` kit.
-
+                    * New world border module, that can be used to create a shifting and/or auto resizing world border.
```

Comment:
We should decide what exactly is going to be listed on this page. Until now, it has just been breaking changes. If this is going to be a complete changelog, then we should have a seperate list of breaking changes.